server/routes/config/licenceConditions.ts

```diff
@@ -0,0 +1,36 @@
+import type { ConditionVersion } from '../../data/licenceClientTypes'
+import { additionalConditionsV1 } from './additionalConditionsV1'
+import { additionalConditionsV2 } from './additionalConditionsV2'
+
+export const standard = {
+  fields: [
+    {
+      additionalConditionsRequired: {
+        responseType: 'requiredYesNo',
+        validationMessage: 'Select yes or no',
+      },
+    },
+  ],
+  modificationRequiresApproval: true,
+}
+
+export const additional = new Map<ConditionVersion, any>([
+  [1, additionalConditionsV1],
+  [2, additionalConditionsV2],
+])
```

Comment:
move to typescript and turned this from an array of the original single set of conditions to a map to allow retrieving conditions by version